src/applications/personalization/account/components/AccountMain.jsx

```diff
@@ -15,26 +14,11 @@ import TermsAndConditions from './TermsAndConditions';
 import facilityLocator from 'applications/facility-locator/manifest';
 
 class AccountMain extends React.Component {
-  constructor(props) {
```

Comment:
since the close button is being removed from the MVI alert, I'm also removing all logic code related to handling closing the alert.